posts/change-data-capture-cloudsql-postgres-with-apache-airflow-1.md

```diff
@@ -0,0 +1,1016 @@
+
+1. To create a new project with the Astronomer CLI, in your terminal, simply create a new directory for your project and then use the Astronomer CLI to scaffold out your project.
+
+    ```bash
+    mkdir <directory-name> && cd <directory-name>
+    ```
+
+    ```bash
+    astro dev init
+    ```
+
+    This will generate the following files in that directory:
+
+    ```bash
+    .
+    ├── dags # Where your DAGs go
+    │   ├── example-dag.py # An example dag that comes with the initialized project
+    ├── Dockerfile # For Astronomer's Docker image and runtime overrides
+    ├── include # For any other files you'd like to include
+    ├── plugins # For any custom or community Airflow plugins
+    ├──airflow_settings.yaml #For your Airflow Connections, Variables and Pools (local only)
+    ├──packages.txt # For OS-level packages
+    └── requirements.txt # For any Python packages
+    ```
+
+2. For version consistency at the time of writing, let's manually set the image our `Dockerfile` uses. Open the `Dockerfile`, change the contents to read:
+
+    ```docker
+    FROM quay.io/astronomer/ap-airflow:1.10.14-1-buster-onbuild
+    ```
+
+3. Next, In the `dags` directory create a new python file with appropriate naming of your choice, such as `change_data_capture.py`. Now (with docker running) start up airflow from the CLI by running:
+
+    ```bash
+    astro dev start
+    ```
+
+4. Once the containers are built and the webserver starts, you can access the UI at  `[localhost:8080](http://localhost:8080)` and login with `admin:admin`.
+
+## Step 5: Create a Connection to GCP
+
+Now we need to add the service account JSON from GCP to Airflow to create a connection. To do so: 
+
+1. In the Airflow UI, open the `Admin -> Connections` tab and create a new connection with the `Connection Type` of `Google Cloud Platform`. 
+2. In the `Scopes` field, add the the following scope:
+
+    ```bash
+    https://www.googleapis.com/auth/cloud-platform
+    ```
+
+3. In the `Keyfile JSON` field, paste the Keyfile JSON you copied from Step 2. 
+
+    ![Connections](../assets/cdc-cloudsql-1/cdc_cloudsql_airflow_2021-01-13_at_4.25.24_PM.png)
+
+Keep the `Conn Id` handy. You will use that in your Airflow DAG to reference this connection. When you're ready, save this connection to finalize your access management. 
+
+## Step 6: Configure Pooling
+
+The next configuration you'll set is a pool. From the [Airflow docs](https://airflow.apache.org/docs/apache-airflow/stable/concepts.html?highlight=pool#pools), `Airflow pools can be used to limit the execution parallelism on arbitrary sets of tasks`, and therefore, using them will enable you to create other parallel jobs while still guaranteeing that your export API tasks will run in serial. 
+
+To create this pool is simple. Just navigate in the top bar to `Admin -> Pools`, then hit the `+` sign to add a new record. Choose a pool name and description to your liking, but choose the number of slots as `1`.
```

Comment:
```suggestion
To create this pool, simply go to the top bar to `Admin -> Pools`, then hit the `+` sign to add a new record. Choose a pool name and description to your liking, but choose the number of slots as `1`.
```

Reply:
@jwitz How about "To create this pool, simply navigate in the top bar to"

Reply:
I'm generally hostile to the word "navigate" since it has a lot of connotations and is much longer than "go". I would simply say "To create this pool, go to `Admin -> Tools`  in the Airflow UI and click the `+` sign". This language is a little more accessible and future proof as well - who know how long those buttons will be in a top bar!